src/connectivity/mod.rs

```diff
@@ -327,7 +331,11 @@ pub fn number_weakly_connected_components(graph: &digraph::PyDiGraph) -> usize {
 #[pyfunction]
 #[pyo3(text_signature = "(graph, /)")]
 pub fn weakly_connected_components(graph: &digraph::PyDiGraph) -> Vec<HashSet<usize>> {
-    conn_components::connected_components(&graph.graph)
+    let return_value: Vec<HashSet<usize>> = connectivity::connected_components(&graph.graph)
+        .into_iter()
+        .map(|res_map| res_map.into_iter().map(|x| x.index()).collect())
+        .collect();
+    return_value
```

Comment:
```suggestion
    connectivity::connected_components(&graph.graph)
        .into_iter()
        .map(|res_map| res_map.into_iter().map(|x| x.index()).collect())
        .collect()
```

Reply:
Done in https://github.com/Qiskit/retworkx/pull/595/commits/d5e5def3556d2221478c59cd334accb372573739.